jekyll/_data/sidenav.yml

```diff
@@ -48,6 +48,8 @@ en:
   children:
     - name: Features
       children:
+        - name: Scheduled Pipelines
```

Comment:
I think maybe adding this a bit further down below the big features, maybe after env vars? @thomasr8

Reply:
@rosieyohannan I will also leave this for now. I'm unsure of what is best here. Since we're suggesting people migrate away from workflows, it makes sense to me to have this above workflows in the navigation.